docs/modules/ROOT/pages/installation/helm.adoc

```diff
@@ -45,9 +45,55 @@ During installation, provide the name of the Kubernetes secret containing the ce
 helm install cerbos cerbos/cerbos --version={app-version} --set=cerbos.tlsSecretName=my-certificate-secret
 ----
 
-If you require advanced features such as automatic certificate reloading, workload identities or mTLS, we recommend deploying a proxy server like link:https://www.envoyproxy.io[Envoy], link:https://github.com/ghostunnel/ghostunnel[Ghostunnel] or link:https://traefik.io[Traefik] as a frontend to the Cerbos server. See the xref:deployment:k8s-sidecar.adoc[Kubernetes sidecar] documentation for an example of deploying Cerbos as a sidecar to Ghostunnel. 
+If you require advanced features such as automatic certificate reloading, workload identities or mTLS, we recommend deploying a proxy server like link:https://www.envoyproxy.io[Envoy], link:https://github.com/ghostunnel/ghostunnel[Ghostunnel] or link:https://traefik.io[Traefik] as a frontend to the Cerbos server. See the xref:deployment:k8s-sidecar.adoc[Kubernetes sidecar] documentation for an example of deploying Cerbos as a sidecar to Ghostunnel.
 ****
 
+== Customizing the manifests
+
+For the sake of simplicity, the Cerbos Helm chart only exposes the settings that are most likely to be changed in a typical deployment scenario. If you want to customize the manifests further, use the post-renderer functionality in Helm to patch the generated manifests before they are applied.
+
+For example, if you want to add a static `loadBalancerIP` to the service generated by the Cerbos Helm chart, you can use Kustomize to patch the service as follows:
```

Comment:
Is this example a bit confusing given that you can achieve this by setting a value?

Reply:
It's due to a complete lack of imagination on my part. Any suggestions for a different field that someone might want to set?

Reply:
How about something a bit more niche like `loadBalancerSourceRanges`?